extensions/reactive-mysql-client/deployment/pom.xml

```diff
@@ -168,29 +185,28 @@
                                             <date>default</date>
                                             <color>cyan</color>
                                         </log>
-                                        <!-- Speed things up a bit by not actually flushing writes to disk -->
+                                        Speed things up a bit by not actually flushing writes to disk -->
```

Comment:
did you mean to remove this comment start?

Reply:
I did not. :)
I think my IDE is trolling me, since at one point I was running that test successfully. Will fix.